skiko/src/commonMain/kotlin/org/jetbrains/skia/Bitmap.kt

```diff
@@ -1053,57 +1053,75 @@ class Bitmap internal constructor(ptr: NativePointer) : Managed(ptr, _FinalizerH
 
 
 @ExternalSymbolName("org_jetbrains_skia_Bitmap__1nGetFinalizer")
+@ModuleImport("skia", "org_jetbrains_skia_Bitmap__1nGetFinalizer")
```

Comment:
Is this annotation needed only for k/wasm? or for 1.8.x k/js too?

Reply:
This annotation needed only for k/wasm.